src/coreclr/vm/customattribute.cpp

```diff
@@ -851,11 +515,7 @@ void COMCustomAttribute::ReadArray(Assembly *pCtorAssembly,
                Module *pModule,
                BASEARRAYREF *pArray)
 {
-    CONTRACTL
-    {
-        THROWS;
-    }
-    CONTRACTL_END;
+    STANDARD_VM_CONTRACT;
```

Comment:
The code is working with raw object references a few lines below. `*pArray = (BASEARRAYREF)AllocatePrimitiveArray`. I do not think this can be STANDARD_VM_CONTRACT that implies PREEMPTIVE_MODE.

Reply:
Boo. I see that. Will update to be COOP explicitly. Thanks.